modules/authentication-remove-kubeadmin.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-authentication.adoc
+
+[id="removing-kubeadmin-{context}"]
+= Removing the kubeadmin user
+
+Once an identity provider has been defined it is recommended to remove
+the `kubeadmin` user for security reasons.
+
+WARNING
+====
+If this procedure is followed before another user is a `cluster-admin`,
+then {product-title} must be reinstalled. It is not possible to undo
+this command once executed.
+====
+
+.Prerequisites
+
+* You must have configured at least one identity provider.
+* You must have added the `cluster-admin` role to a user.
```

Comment:
Do we have a clear topic for explaining how to add the `cluster-admin` role to a user? If so, I'd move this prereq to the assembly and add a link out.

Reply:
We don't presently. We mention this specific role in several places, but there are no examples of adding it to a user. I've filed [BZ-1694774](https://bugzilla.redhat.com/show_bug.cgi?id=1694774) to include this example.